contrib/verify-commits/trusted-sha512-root-commit

```diff
@@ -0,0 +1 @@
+b00ba6251f71fa1edaabdf809514e1bc3c67862e
```

Comment:
This commit is referring to a Bitcoin commit, so it's useless for us. Should change this to a Dash commit in the future.